Modules/bb8/_main_.env

```diff
@@ -32,10 +32,17 @@
 #### HEADER BLOC ####
 
 source $T4D_SOURCE_PATH/Git/_main_.env
-source $T4D_SOURCE_PATH/BitBucket/_main_.env
-source $T4D_MODULES_PATH/bb8/bb8-utils.env
+case $T4D_BB8_MODE in 
```

Comment:
Please add Default value in Modules/bb8/_main_.env 

before bb8() definition
`T4D_BB8_MODE="${T4D_BB8_MODE:-bitbucket}"`